### PR TITLE
Loader

### PR DESCRIPTION
**Pull request by vips28**

### CODE REVIEW

**Comment by AniVerma17** [comment]
Provide a demo preview for this.

File: index.html [hunk 1]
```diff
@@ -1,5 +1,35 @@
 <!DOCTYPE html>
 <html>
+<<<<<<< HEAD
```
Comment (AniVerma17) [inline index.html]: Remove this line, and also lines 32 and 53.

**Comment by vips28** [comment]
https://vips28.github.io/Pyladies-delhi-website/

**Comment by vips28** [comment]
@AniVerma17 Please Check

**Comment by AniVerma17** [comment]
Enhance the styling and layout, and the dots are too small.

**Comment by vipulgupta2048** [comment]
@vips28 Here are some tips to save your fellow teammates, some time. 
1. Write a good descriptive pull request header. People should know exactly what this contains. I am stating this for the future. Also, a PR description is a must until and unless the header clearly states everything. 
2. Squashing commits in git. Learn it. [@utkarsh2102  Don't merge it until that happens.] Take help from others. 
3. @AniVerma17 Good catch that you found a bug on line 32 and 53 but a description along with it. Helps. 
@AniVerma17  @utkarsh2102  @anuja

**Comment by utkarsh2102** [comment]
@vips28, try to lessen the number of commits. You've made 10 commits in such small intervals.
This is very chaotic sometimes. Try to minimise the number of commits you make.

**Comment by AniVerma17** [comment]
- Loader is good, but that PyLadies text image is a bit blurry & doesn't look good, I suggest to remove it.
- Remove horizontal overflow from the page.
- The lady image becomes too small on mobile screens, so make its size responsive too.

**Comment by vips28** [comment]
Thanks @AniVerma17 @vipulgupta2048 @utkarsh2102 for your feedback.
I will totally change what you have told me

**Comment by AniVerma17** [comment]
Please resolve the conflicts, and squash your commits into one.

**Comment by vipulgupta2048** [comment]
@utkarsh2102  Test it on Github pages. Merge if working.

File: index.html [hunk 2]
```diff
@@ -1,166 +1,43 @@
...
+  img{
+    max-width: 100%;
+    height: auto;
+    border-radius: 25%;
+    align-items: center;
+  }
+  </style>
+  <body background-color="black">
+    <div class="img" style="text-align: center">
+    <img src="pylady_geek.png" alt="Pyladies" height="30%" width="30%">
```
Comment (AniVerma17) [inline index.html]: Remove height and width attributes here. It is a deprecated practice to define these in HTML for images, plus it affects the height and width defined in CSS for the same.

File: index.html [hunk 3]
```diff
@@ -1,166 +1,43 @@
...
+  </head>
+  <style media="screen" type="text/css">
+  div{
+    margin: auto;
+    align-self: auto;
+    style="align-items: center"
+  }
+  img{
+    max-width: 100%;
+    height: auto;
```
Comment (AniVerma17) [inline index.html]: This isn't effective until the height and width of parent div is defined in CSS.

**Comment by utkarsh2102** [comment]
@vips28, there are changes to be made (as told by @AniVerma17) and you're also supposed to resolve the merge conflict, too.

**Pull request closed without merging**